spec/Conversions.tex

```diff
@@ -405,7 +405,7 @@ \subsection{Explicit Enumeration Conversions}
 \index{conversions!enumeration}
 \index{conversions!explicit!enumeration}
 
-Explicit conversions are allowed from any enumerated type to any
+Explicit conversions are allowed from any enumerated type to
 \chpl{bytes} or \chpl{string} and vice-versa, and include \chpl{param} conversions.
```

Comment:
I'm reading the sentence as written as "to any `bytes` or `string` type".  It doesn't make sense to me to speak of "any string type" or "any bytes type", since there's only one of each, but I could be missing something.

Reply:
@e-kayrakli: Not sure if you saw this, but it looks right to me.  Can you double-check as well, and merge if you agree?

Reply:
@bradcray -- thanks for the ping. That looks right to me, as well. I'll merge